### PR TITLE
Error Popup show and SelectedVariations Data Refactor (DEL-12124)

### PR DESCRIPTION
**Pull request by Harshverma2308**

### CODE REVIEW

File: src/componets/ProductVariations.jsx [hunk 1]
```diff
@@ -1,215 +1,352 @@
...
+  const handleChange = (e) => {
+    const { value, name } = e.target;
+    setCustomSize((prev) => ({
+      ...prev,
+      [name]: Number(value),
+    }));
+  };
+
+  const handleHeight = (variant) => {
+    console.log(selectedVariations[variant?.title].configs.minHeight)
```
Comment (devbyhitesh) [inline src/componets/ProductVariations.jsx:68]: Remove this console

File: src/hooks/useProductVariables.js [hunk 2]
```diff
@@ -2,108 +2,156 @@ import { useEffect, useState } from "react";
...
-            .filter(item => !item.filters.length)
-            .map(item => {
-                const hasDefaultValue = item.items.findIndex((item) => item.default == 1);
-                setSelectedVariations(prevValue => ({
-                    ...prevValue,
-                    [item.title]: hasDefaultValue ? item.items[hasDefaultValue] : item.items[0]
-                }))
-            })
-    }, [initialVariations]);
+  console.log(variations, "::");
```
Comment (devbyhitesh) [inline src/hooks/useProductVariables.js:10]: Remove this console

**Comment by devbyhitesh** [comment]
https://gist.github.com/devbyhitesh/2081c421609b9ae525a1714fcff53f63
Let's introduce the prettier in the code base.

**Comment by devbyhitesh** [comment]
cc @dishantagnihotri for final approval

**Review by devbyhitesh — APPROVED** [review]

**Review by dishantagnihotri — APPROVED** [review]
Unable to verify due to eslint changes. Looks good though.

**Pull request merged**